### PR TITLE
Fixes notice

### PR DESCRIPTION
**Pull request by annando**
See https://github.com/friendica/friendica/issues/11218#issuecomment-1099536315

### CODE REVIEW

File: src/Module/Item/Compose.php [hunk 1]
```diff
@@ -135,7 +135,7 @@ protected function content(array $request = []): string
 
 		$contact = Contact::getById($a->getContactId());
 
-		if ($this->config->get(local_user(), 'system', 'set_creation_date')) {
+		if (DI::config()->get(local_user(), 'system', 'set_creation_date')) {
```
Comment (nupplaphil) [inline src/Module/Item/Compose.php:138]: Please overwrite the `__construct()`or instead and include the Config interface instead
@MrPetovan - tbh I'm doing hard remembering the right interface names, before the restructure, it was easy - `IConfig`, now I do have to think if it was `ICanConfig` or `IHandleConfigs`, ... Maybe we should discuss about it

**Review by nupplaphil — CHANGES_REQUESTED** [review]

File: src/Module/Item/Compose.php (hunk 1 above)
Comment (annando) [reply src/Module/Item/Compose.php:138]: `DI` was used before in this file, so I think that this is a quick solution for an error.

Comment (nupplaphil) [reply src/Module/Item/Compose.php:138]: Ok, but please add a follow-up removing the `DI::` calls..

Comment (annando) [reply src/Module/Item/Compose.php:138]: I have to have a look at where to copy it from and then have to implement it without crashing the system :-)

Comment (MrPetovan) [reply src/Module/Item/Compose.php:138]: @nupplaphil I'm sorry about the inconvenience, I still believe that more descriptive names are better, and we can keep discussing it.

**Pull request merged**